src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteClient.java

```diff
@@ -66,6 +70,47 @@ public static MapRouletteClient instance(final MapRouletteConfiguration configur
         return null;
     }
 
+    /**
+     * This method writes challenge ids to a file.
```

Comment:
Do you mind adding a note on the format of the file?

Reply:
Done